### ISSUE TITLE
No need to load all assemblers

### ISSUE THREAD

**Issue by lskatz** [tseemann/shovill#137]
Sorry if this is a nonissue but I _think_ that Shovill requires all assemblers (skesa, spades, megahit) to be loaded in `check_deps`.  If all dependencies are not loaded, then it calls `err` and exits with an exit code of 1.

Is there a way to remove the dependency check for assemblers that are not used?  Is that something I could make a pull request for?

**Comment by tseemann** [comment]
It's to ensure people have installed their conda or brew package properly, and discourage people manually installing it.

Of course it doesn't need to be that way. I don't have a good track record of accepting PRs unfortunately, I get scared.

**Comment by tseemann** [comment]
@lskatz I agree a dynamic system would be good, wher eit even changes the available parameters and/or explains how you can 'extend' the capabilities.

but with modern conda, just install them all I reckon.
